src/views/TodoList.jsx

```diff
@@ -1,12 +1,28 @@
 import { Container } from "@mui/material";
 import AddTask from "../components/AddTask";
 import TaskList from "../components/TaskList";
+import { useState } from "react";
 
 const TodoList = () => {
+  const [todoList, setTodoList] = useState([]);
+
+  const handleAddTodoList = (title) => {
+    setTodoList([
+      ...todoList,
+      {
+        id: todoList.length + 1,
+        title: title,
+        done: false,
+      },
+    ]);
+  };
+
+  console.log(todoList);
```

Comment:
Same here, remove consolog and wrap the callback with `useCallback` (MUST read document)